test/test_c10d.py

```diff
@@ -2370,6 +2370,57 @@ def check_no_grads():
         # No parameter should have their gradient set.
         check_no_grads()
 
+    @skip_if_not_multigpu
+    @skip_if_not_nccl
+    def test_accumulate_gradients(self):
```

Comment:
Shall we add a comment to say that this would only work when this is a single model replica per process.